tools/wrapper/retrace.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <errno.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+
+/*
+ * print usage
+ */
+
+static void usage(const char *prog_name)
+{
+	fprintf(stderr, "Usage: %s -l <path of libretrace.so> <command line>\n",
+			prog_name);
+	exit(1);
+}
+
+/*
+ * fork process
+ */
+
+static void fork_process(int argc, char **argv)
```

Comment:
`fork_process` will be more useful if it takes a single args parameter suitable for `execvp`. It turns out that `argv` in `main` is already a null terminated array ...